app.config.js

```diff
@@ -11,10 +14,12 @@ module.exports = {
 
   // List of chainIds which metadata cache queries will return by default.
   // This preselects the Chains user preferences.
-  chainIds: [1, 137, 56, 246, 1285],
+  chainIdsProduction: chainIdsProduction,
```

Comment:
do we really need all this split-up introduced here? On first look, seems this is only used in `getVeChainIds()` which has more hardcoded chainIds. So would suggest keeping app config as before, and rather use new setting dedicated for ve stuff, like `veChainId` or so.